src/main/java/steps/gui/openforms/OpenFormsSteps.java

```diff
@@ -44,8 +37,8 @@ public void open_pagina() {
     public void login_via_digid(User user) {
         openFormsPage.buttonAccepteerCookies.click();
         get_tekst("Inloggen met DigiD").click();
-        digidLoginSteps.login_via_digid(user);
-        openFormsPage.textlabelHeaderH1
+        digidLoginSteps.Login(user);
+        openFormsPage.uitloggenDigidButton
```

Comment:
dit zou volgens mij ook moeten werken met gewoon openFormsPage.uitloggenDigidButton.waitFor()

Reply:
klopt inderdaad, zullen we het veranderen naar openFormsPage.uitloggenDigidButton.waitFor()? 
Ik heb hier alleen de textlabelHeaderH1 veranderd naar uitloggenDigidButton